proto/xudrpc.proto

```diff
@@ -443,7 +449,7 @@ message Peer {
   string address = 1 [json_name = "address"];
   // The node pub key to uniquely identify this peer.
   string node_pub_key = 2 [json_name = "node_pub_key"];
-  // A map of ticker symbols to lnd pub keys for this peer 
+  // A map of ticker symbols to lnd pub keys for this peer
```

Comment:
Good catch on the whitespace, but while we're changing this line we might as well add a period at the end to make it consistent with the other comments.